### PR TITLE
Bug 1184510 - Change some more UI references to 'result set'

### PR DESCRIPTION
**Pull request by KWierso**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/2137)

### CODE REVIEW

File: ui/userguide.html [hunk 1]
```diff
@@ -559,7 +559,7 @@
               <table id="queryparams">
                 <tr>
                   <td><span class="queryparam">nojobs</span></td>
-                  <td>Load result sets and revisions without loading any job results.</td>
+                  <td>Load pushes and revisions without loading any job results.</td>
```
Comment (KWierso) [inline ui/userguide.html]: I'd like another opinion on changing this one. Can this be rephrased instead of just s/result sets/pushes/?

Comment (KWierso) [inline ui/userguide.html]: Maybe "Load push metadata and revisions without..."?

Comment (wlach) [inline ui/userguide.html]: How about just `Load pushes without loading any job results`? I think the revisions/metadata parts are implied.

File: ui/js/directives/treeherder/resultsets.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ treeherder.directive('thAuthor', function () {
             scope.authorName = userTokens[0].trim();
             scope.authorEmail = email;
         },
-        template: '<span title="View resultsets for {{authorName}}: {{authorEmail}}">' +
+        template: '<span title="View pushes for {{authorName}}: {{authorEmail}}">' +
```
Comment (edmorley) [inline ui/js/directives/treeherder/resultsets.js]: What do you think about "View pushes by {{authorEmail}}"? I think the author name is redundant (after all it's the email that goes in the URL filter param).

File: ui/userguide.html (hunk 1 above)
Comment (edmorley) [inline ui/userguide.html]: I like Will's suggested wording :-)

**Pull request merged**